torch/_dynamo/config.py

```diff
@@ -425,8 +425,6 @@ def default_debug_dir_root():
 # correctness of custom ops.
 only_allow_pt2_compliant_ops = False
 
-capture_autograd_function = True
```

Comment:
You should not delete the flag. The idea is to keep the flag here, but make it a no-op.